CHANGELOG.md

```diff
@@ -2,9 +2,69 @@
 
 ## Unreleased
 
-## [1.3]
+## 2.0
 
-### [1.3.0] - 2021-07-26
+### 2.0.0 - 2022-08-30
+
+Make sure to read about the breaking changes and the release highlights below!
+
+#### Breaking changes
+
+If you are using chartpress to publish development releases should be aware that
+the suffix appended to chart versions and image tags is changed to look like
+`1.2.3-0.dev.git.n.hash`, where the following things has changed:
+
+- `-0.dev` is appended by default for non pre-release versions.
+- `n` is the number of commits on the branch, where it previously was the number
+  of commits since the last tag on the branch.
+
+#### Release highlights
+
+- The option `baseVersion` is added to chart configuration in chartpress.yaml,
+  for more details see the [README.md section on controlling development
+  versions](https://github.com/jupyterhub/chartpress#controlling-development-versions).
+
+#### New features added
+
+- use baseVersion config to set the base version for development releases (supersedes useChartVersion config) [#179](https://github.com/jupyterhub/chartpress/pull/179) ([@minrk](https://github.com/minrk))
+- (reverted) Add `useChartVersion` and change appended version suffix (now like `1.2.3-0.dev.git.3.h123`) [#150](https://github.com/jupyterhub/chartpress/pull/150) ([@minrk](https://github.com/minrk))
+
+#### Enhancements made
+
+- (reverted) accept --tag arg in --reset [#152](https://github.com/jupyterhub/chartpress/pull/152) ([@minrk](https://github.com/minrk))
```

Comment:
I think it's confusing to list this as an enhancement, even with the `reverted` note, because it's not actually an enhancement that exists. I think _removing_ the two PRs (this one and #150) from the list provides a better representation of the release.

If you want to keep them listed, I'd move reverted/superseded PRs to maintenance where they are (rightly) usually ignored.

Reply:
Agree!

I moved them to the maintenance section as I find that having a list of "this could have caused the bug I'm observing" is often very helpful.